src/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './media/local-camera-track';
 export * from './media/local-display-track';
 export * from './media/local-microphone-track';
 export * from './media/local-track';
+export * from './media/local-video-track';
```

Comment:
LocalComputerAudioTrack needs to be exported here too I think

Reply:
done